public/components/custom_panels/sub_components/visualization_panel.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import {
+  EuiButton,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiPopover,
+  EuiText,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+
+// Visualization Panel module allows view added viz modules.
+
+type Props = {
+  visualizationId: string;
+  visualizationTitle: string;
+  fromTime?: string;
+  toTime?: string;
+};
+
+export const VisualizationPanel = ({
+  visualizationId,
+  visualizationTitle,
+  fromTime,
+  toTime,
+}: Props) => {
+  const [isPopoverOpen1, setIsPopoverOpen1] = useState(false);
```

Comment:
Just curious for why these variables were named with an 1 in the end?

Reply:
I used this from EUI examples. Probably stuck with the same variable name, will change this.